pkg/azdiskschedulerextender/azdiskschedulerextender_test.go

```diff
@@ -0,0 +1,562 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"testing"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/klog/v2"
+	schedulerapi "k8s.io/kube-scheduler/extender/v1"
+	v1alpha1Client "sigs.k8s.io/azuredisk-csi-driver/pkg/apis/azuredisk/v1alpha1"
+	fakeClientSet "sigs.k8s.io/azuredisk-csi-driver/pkg/apis/client/clientset/versioned/fake"
+	fakeClientSetInterface "sigs.k8s.io/azuredisk-csi-driver/pkg/apis/client/clientset/versioned/typed/azuredisk/v1alpha1/fake"
+)
+
+var (
+	handlerFilter     = http.HandlerFunc(handleFilterRequest)
+	handlerPrioritize = http.HandlerFunc(handlePrioritizeRequest)
+)
+
+func TestFilterAndPrioritizeRequestResponseCode(t *testing.T) {
```

Comment:
Can we simulate a 5000 node cluster and then try to filter\priortize (50000/3)*100 pods with a total of 30000 disks. We just want to see how much time it would take, we can have 15-20 (may be have a configurable option) concurrent filter\priortize going in.
This will give us a hint on code's performance.